src/core/IC3Initializer.ts

```diff
@@ -1,3 +1,5 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
Should we disable this? Could we use unknown instead?

Reply:
Good point. A good portion of it is fixed. There's still some, but they're not as simple as just changing the typing on one file.